src/averageAges.js

```diff
@@ -15,13 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => (person.sex === 'm')
+    && (century ? century === Math.ceil(person.died / 100) : true));
+
+  const ageOfDeath = men.map(man => man.died - man.born);
+
+  return getAverageAge(ageOfDeath);
 }
 
+// let men;
```

Comment:
don't leave unnecessary comments in your code

Reply:
Done I already revomed them!